py/picca/fitter2/chi2.py

```diff
@@ -27,6 +26,7 @@ def __init__(self,dic_init):
             self.verbosity = dic_init['verbosity']
 
         if 'fast mc' in dic_init:
+            self.seedfast_mc = dic_init['fast mc']['seed']
```

Comment:
I disagree with forcing the user to choose a seed for the `fast mc`, the code should handle it correctly internally